### DISCUSSION TITLE
New model not working.

### DISCUSSION THREAD

**Discussion by Papa-Yankee** [EdgeTX/edgetx#4782] [Q&A]
Hello everyone.
I'm in a lot of trouble. I switched two weeks ago to Edge TX on my TX16S, since Open TX. The models present at the time of the switchover work perfectly well.
I'm now entering a new model (a Baron that I'm just finishing building). Being the slacker that I am, I foolishly duplicated an existing model. I changed the photo (fundamental!), and plugged in a receiver.
Everything seemed to be going well until I move the sticks. Randomly, the receiver's LED turns red, the servos lock in position, and I can't regain control until a variable delay has elapsed.
Edge advised me to update the firmware of the 4-in-1 module, which I did, but no change.
I tested with two different receivers (RX8R pro), same symptoms. I re-tested my trainer, which still works perfectly. I compared the two models under Companion, and I can't see any difference that might explain the phenomenon....
I also tried to create a model ex nihilo from the transmitter, but identical symptoms...
If anyone has any ideas, I'd love to hear them, because right now, I'm at a loss...

Thanks for reading.

**Comment by Papa-Yankee** [comment]
Problem solved. It was caused by a deficient power supply.

**Comment by pfeerick** [comment]
Glad to hear you solved it. Reading the symptoms again, it would probably
have been that or servos that needed checking.
